### ISSUE TITLE
Exclusion of branches to the regexp required for the commit message

### ISSUE THREAD

**Issue by glndgrgd** [tomasbjerre/simple-bitbucket-commit-checker#62]
Hi,

Is there a way to exclude some branches from being checked? We currently have to audit the commit messages for specific branches only.

**Comment by tomasbjerre** [comment]
Yes. There is a regexp to check only branches matching a regexp.

Den 10 nov. 2016 20:08 skrev "glndgrgd" <notifications@github.com>:

**Comment by glndgrgd** [comment]
However, our users will need to input more than 1 branch

**Comment by tomasbjerre** [comment]
You can have "or" as | in regexp.

Den 10 nov. 2016 21:34 skrev "glndgrgd" <notifications@github.com>:

**Comment by tomasbjerre** [comment]
Maby somthing like https://jex.im/regulex/#!embed=false&flags=&re=%5Erefs%2Fheads%2F(branch1%7Cbranch2%7Cbranch3)%24
